xExtension-YouTube/configure.phtml

```diff
@@ -1,38 +1,37 @@
-
-<form action="<?php echo _url('extension', 'configure', 'e', urlencode($this->getName())); ?>" method="post">
-	<input type="hidden" name="_csrf" value="<?php echo FreshRSS_Auth::csrfToken(); ?>" />
+<form action="<?= _url('extension', 'configure', 'e', urlencode($this->getName())) ?>" method="post">
+	<input type="hidden" name="_csrf" value="<?= FreshRSS_Auth::csrfToken() ?>" />
 	<div class="form-group">
 
-		<label class="group-name" for="yt_height"><?php echo _t('ext.yt_videos.height'); ?></label>
+		<label class="group-name" for="yt_height"><?= _t('ext.yt_videos.height') ?></label>
 		<div class="group-controls">
-			<input type="number" id="yt_height" name="yt_height" value="<?php echo FreshRSS_Context::$user_conf->yt_player_height; ?>" min="50" data-leave-validation="1">
+			<input type="number" id="yt_height" name="yt_height" value="<?= $this->getHeight() ?>" min="50" data-leave-validation="1">
 		</div>
 
-		<label class="group-name" for="yt_width"><?php echo _t('ext.yt_videos.width'); ?></label>
+		<label class="group-name" for="yt_width"><?= _t('ext.yt_videos.width') ?></label>
 		<div class="group-controls">
-			<input type="number" id="yt_width" name="yt_width" value="<?php echo FreshRSS_Context::$user_conf->yt_player_width; ?>" min="100" data-leave-validation="1">
+			<input type="number" id="yt_width" name="yt_width" value="<?= $this->getWidth() ?>" min="100" data-leave-validation="1">
 		</div>
-
-		<label class="group-name" for="yt_show_content"><?php echo _t('ext.yt_videos.show_content'); ?></label>
+		
+		<label class="group-name" for="yt_show_content"><?= _t('ext.yt_videos.show_content') ?></label>
 		<div class="group-controls">
-			<input type="checkbox" id="yt_show_content" name="yt_show_content" value="1" <?php if (FreshRSS_Context::$user_conf->yt_show_content != '') { echo 'checked'; }; ?>>
+			<input type="checkbox" id="yt_show_content" name="yt_show_content" value="1" <?= $this->getShowContent() ? 'checked' : '' ?>>
 		</div>
 
-		<label class="group-name" for="yt_nocookie"><?php echo _t('ext.yt_videos.use_nocookie'); ?></label>
+		<label class="group-name" for="yt_nocookie"><?= _t('ext.yt_videos.use_nocookie') ?></label>
 		<div class="group-controls">
-			<input type="checkbox" id="yt_nocookie" name="yt_nocookie" value="1" <?php if (FreshRSS_Context::$user_conf->yt_nocookie != '') { echo 'checked'; }; ?>>
+			<input type="checkbox" id="yt_nocookie" name="yt_nocookie" value="1" <?= $this->getUseNoCookie() ? 'checked' : '' ?>>
 		</div>
```

Comment:
If you still have some motivation, you can improve the look of the checkboxes by moving the labels on the right:

```html
		<div class="group-controls">
			<label class="checkbox" for="yt_show_content">
				<input type="checkbox" id="yt_show_content" name="yt_show_content" value="1" <?= $this->getShowContent() ? 'checked' : '' ?>>
				<?= _t('ext.yt_videos.show_content') ?>
			</label>
		</div>

		<div class="group-controls">
			<label class="checkbox" for="yt_nocookie">
				<input type="checkbox" id="yt_nocookie" name="yt_nocookie" value="1" <?= $this->getUseNoCookie() ? 'checked' : '' ?>>
				<?= _t('ext.yt_videos.use_nocookie') ?>
			</label>
		</div>
```

It looks like this 

![Capture d’écran de 2020-01-17 09-17-09](https://user-images.githubusercontent.com/1436309/72595474-303d4780-390a-11ea-8a78-e401f7d281c7.png)

Reply:
Done ;)